packages/flutter/lib/src/material/drawer.dart

```diff
@@ -5,6 +5,8 @@
 import 'package:flutter/foundation.dart';
 import 'package:flutter/widgets.dart';
 
+import 'package:flutter/material.dart';
```

Comment:
Don't forget to fix this import 📁

Reply:
But I get like 100 errors if I take it out... How do I know which specific files to include?

Reply:
On the auto import suggestion in intellij you should see an option that imports from '/material/src` - just pick those and then make them relative.

Reply:
Looks like I didn't need that import, must have not been filtering for the errors in the current file before.